### ISSUE TITLE
x-sb-psc - endpoint issue - dns

### ISSUE THREAD

**Issue by pysiok1983** [apigee/terraform-modules#31]
Hi,
I am using your code for psc/dns but for more than one dns peering to the apigee I am not able to use 7.x.x.x A record on apigee. Is there any limitation?

**Comment by pysiok1983** [comment]
I solved it.

**Comment by danistrebel** [comment]
@pysiok1983 before I start making any assumptions can you please be more specific about:

1. The configuration you performed
2. The expected behavior
3. The observed behavior

**Comment by pysiok1983** [comment]
Hi, 
I am going to use more than one private zones that will be peered to APIgee (DNS peering). When I use only one zone (.local) all is fine and I am able to add a DNS record that is used within APIGee as a backend. When I configured more than one zone the APIgee is not able to resolve the name configured as an A record in one of the zones. 
1. More than one DNS private zones should peer to APIGee. 
2. Each record across the zones should be available by APIgee. 
3. As I mentioned seems that I am able to use only one private zone.

**Comment by danistrebel** [comment]
Yeah there must be something wrong with your configuration. DNS peering supports peering with multiple suffixes and Apigee is able to use all configured A records within each.

You should see the peerings when you run something like this:

```sh
$ gcloud services peered-dns-domains list --network apigee-network


NAME: barinternal
DNS_SUFFIX: bar.internal.

NAME: foointernal
DNS_SUFFIX: foo.internal.
```

**Comment by pysiok1983** [comment]
Let me check this again.

**Comment by danistrebel** [comment]
Closing this as it seems resolved.
